grails-app/controllers/rgms/publication/TeseController.groovy

```diff
@@ -14,7 +14,11 @@ class TeseController extends ThesisOrDissertationController {
     }
 
     def save() {
-        saveThesisOrDissertation("Tese", params)
+        String file = params['file']
+        def format = file.substring(file.indexOf('.')+1,file.size())
+        if(format == 'doc' || format == 'pdf') {
```

Comment:
(mfl3) Aqui não seria melhor criar um método genérico para validar o formato do arquivo? E se no futuro for necessário alterar a definição de "formato válido" (e.g se eu passar a aceitar arquivos no formato .odt)?